test/new_relic/agent/configuration/security_policy_source_test.rb

```diff
@@ -177,33 +177,10 @@ def test_message_parameters_disabled
           end
         end
 
-        def test_job_arguments_enabled
```

Comment:
Deleted because these tests are only testing to make sure the new removed configs worked correctly. Since it is now treated like any other attribute, the tests for `attribute.include` now cover these situations.